doc/03-Configuration.md

```diff
@@ -34,17 +29,37 @@ port:
 
 `443,5665-5669`
 
-Scan jobs can be executed using the `icingacli x509 scan` CLI command. The `--job` option is used to specify the scan
-job which should be run:
+Additionally, each job may also exclude specific **hosts** and **IP** addresses from scan. These hosts won't be scanned
+when you run the [scan](04-Scanning.md#scan-command) or [jobs](04-Scanning.md#scheduling-jobs) command. Excluding an
+entire network and specifying IP addresses in CIDR format will not work. You must specify a comma-separated concrete
+**IP** and **host CN**, e.g:
 
-```
-icingacli x509 scan --job lan
-```
+`192.0.2.2,192.0.2.5,icinga.com`
+
+### Job Schedules
+
+Schedules are `cron` and [rrule](https://www.rfc-editor.org/rfc/rfc5545) based configs used to run periodically
+at the given interval. Every job is allowed to have multiple schedules that can be run independently of each other.
+Don't worry, you don't need to know anything about rrule to create **rrule** based schedules. All you need to do is
```

Comment:
Why mention **rrule** at all? Is that shown somewhere in the frontend? I would speak of "repeat patterns", "rule based", or something like that.